src/Form.js

```diff
@@ -284,7 +284,7 @@ class Form extends React.Component {
 
             <div />
             <h2>Target</h2>
-            { this.LabelledField("Network", "network", this.Select("network", ["main", "demo", "test"])) }
+            { this.LabelledField("Network", "network", this.Select("network", ["main", "demo", "test", "testv4"])) }
```

Comment:
I don't think we should expose testV4 in the UI

Reply:
PS I merged and published the player, update the package.json here to require player >= 1.0.78